lib/features/onboarding/widget/login_with_card.dart

```diff
@@ -0,0 +1,31 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:quranpally/shared/widgets/app_image.dart';
+
+class LoginWithCard extends StatelessWidget {
+  const LoginWithCard({
+    super.key,
+    required this.image,
+    required this.onTapped,
+  });
+  final String image;
+  final VoidCallback onTapped;
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: onTapped,
+      child: Container(
+        height: 44.h,
+        width: 60.w,
+        decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(10),
+            color: const Color(0xFFECECEC)),
```

Comment:
Move this to a class in the theme folder
Like theme/colors/app_colors.dart
And any other color you may be using